### PR TITLE
Commit for the first review

### PR DESCRIPTION
**Pull request by massaillon**
The purpose of this commitment is for practicing the review flow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# First-review
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (massaillon) [inline README.md:5]: Hi
please review your README file, I suggest you use a professional README file template below
 https://github.com/microverseinc/readme-template

Comment (massaillon) [reply README.md:5]: 1)There is no sign of any Linters installed, I suggest you install all the linters 

      * you can follow the below link for the linters installation and use
        Once this is done do the first check on your local PC before doing a request once there are no linters     errors please do a pull request
        
        https://github.com/microverseinc/linters-confighttps://github.com/microverseinc/linters-config

Comment (massaillon) [reply README.md:5]: We noticed that you have unused files.DS_Store in your directory, we suggest you add these unused files to the .gitignore file

**Review by massaillon — COMMENTED** [review]
Changes required
Please check my below comment to see the changes you have to make
Thanks

**Review by vic778 — APPROVED** [review]
Hi @massaillon , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**